test/functional/migrations/generate-command/command.ts

```diff
@@ -11,10 +11,8 @@ describe("migrations > generate command", () => {
         schemaCreate: false,
         dropSchema: true,
         entities: [Post, Category],
-        logging: true,
```

Comment:
Can you drop the changes to this file?

Reply:
I will do that. Can I know the reason?

Reply:
Because the changes were mostly stylistic - so removing them reduces the footprint of this code change.

I'll separately from this drop the logging flag from testing connections.